app/features-json/build_json_controller.rb

```diff
@@ -85,7 +87,7 @@ class BuildJSONController < APIController
       # if the server was restarted, we're gonna end here in this code block
       build_log_artifact = current_build.artifacts.find do |current_artifact|
         # We can improve the detection in the future, to actually mark an artifact as "default output"
-        current_artifact.type.include?("log") && current_artifact.reference.end_with?("fastlane.log")
+        current_artifact.type.include?("log") && current_artifact.reference.end_with?("runner.log")
```

Comment:
this was changed to `runner.log` because I wanted to make sure we don't confuse this log with the `fastlane.log` which is part of the archive bundle. That log does not include things like the git checkout, and other setup. Also it is raw ANSI text, and not rows of JSON records, which is what `runner.log` is.